test/cli/log_redaction_test.go

```diff
@@ -0,0 +1,41 @@
+package cli
+
+import (
+	"strings"
+	"testing"
+)
+
+func TestLogRedaction_CommandConfigs(t *testing.T) {
+	secretValue := "topsy-kretts"
+	tests := []struct {
+		name       string
+		args       []string
+		env        map[string]string
+		assertions []traitAssertion
```

Comment:
should these be removed until they are used?

Reply:
I can do that -- this was lifted from syft and neglected to remove the extra fields